static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -755,11 +761,17 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
             }
         }
     }
+
+    // Single line limited variant, stacked variant
+    &.tertiary__variant-four,
```

Comment:
Shouldn't variants be modifiers? I haven't seen the full markup but that would seem to make sense.